app/models/manageiq/providers/ibm_cloud/inventory/parser/power_virtual_servers.rb

```diff
@@ -242,20 +242,22 @@ def flavors
         :name    => value.type
       )
     end
-    collector.sap_profiles.each do |value|
-      description = ''
-      if value.certified
-        description = 'certified'
+    if collector.cloud_instance.capabilities.include?('sap')
```

Comment:
In order to memoize this I think we should do:
```
def cloud_instance_capabilities
  @cloud_instance_capabiliies ||= cloud_instances_api.pcloud_cloudinstances_get(cloud_instance_id)
end
```

In the collector so that other callers can make use of the capabilities without having to make another API call.

Reply:
:+1: on memoization. Is there an advantage in keeping only the 'capabilities' and excluding the rest of the response data? I think it's likely we'll be using some of the other data. (@Kuldip-Nanda noticed an issue with the cloud manager 'region' value that could possibly be fixed with this).

Reply:
Nope if there's other good info in that response let's save that too 👍

Reply:
Done!